vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -2391,9 +2392,17 @@ func TestStandardExecutorConvertSubnetToL1Tx(t *testing.T) {
 		)
 	)
 
+	testAddresses := make([]ids.ShortID, 0)
+	for _, testKey := range genesistest.DefaultFundedKeys {
+		testAddresses = append(testAddresses, testKey.Address())
+	}
```

Comment:
I don't think we should ever `make(XXX, 0)`.
```suggestion
	testAddresses := make([]ids.ShortID, len(genesistest.DefaultFundedKeys))
	for i, testKey := range genesistest.DefaultFundedKeys {
		testAddresses[i] = testKey.Address()
	}
```